qml/Launcher/LauncherPanel.qml

```diff
@@ -90,16 +91,21 @@ Rectangle {
             color: UbuntuColors.orange
             readonly property bool highlighted: root.highlightIndex == -1;
 
-            Image {
+            Icon {
                 objectName: "dashItem"
                 width: parent.width * .6
                 height: width
-                sourceSize.width: width
-                sourceSize.height: height
                 anchors.centerIn: parent
-                source: "graphics/home.svg"
+                source: {
+                    if (root.showCloseIcon) {
+                        return "/usr/share/icons/suru/actions/scalable/back.svg"
+                    }
+                    return "graphics/home.svg"
```

Comment:
Please put ending semicolons for statements in JS blocks, for consistency (the rest of the JS added by this diff includes them). :)